nautobot/extras/templates/extras/configcontextschema.html

```diff
@@ -54,3 +46,14 @@
                 </div>
             </div>
 {% endblock content_right_page %}
+
+
+{% block javascript %}
+<script type="text/javascript">
+    // Set config context schema as selected tab on page load
+    if (location.pathname == "{{ object.get_absolute_url }}" && !location.search.includes("tab")){
+        $(`.nav.nav-tabs a[href="{{ object.get_absolute_url }}#main"]`).tab("show")
+    }
```

Comment:
This feels hacky to me. Why doesn't the main tab get shown by default? Would a better fix be to change something about `ConfigContextSchemaView.get_extra_context` perhaps?

Reply:
Thanks